README.md

```diff
@@ -38,6 +43,22 @@ module.exports = function(environment) {
 };
 ```
 
+### Using the service
+
+The service allows you to react to an app update in a more programmatic manner, e.g. 
+you could force reload the app. The service emits an `update` event once an update
+has been detected. Here is an example of an instance-initializer that reloads the app
```

Comment:
I don't think we should encourage initializers :/

Reply:
This code could be configured in the application's before model or init method?

Reply:
Makes sense. Updated the example!